aten/src/ATen/native/native_functions.yaml

```diff
@@ -3056,6 +3056,14 @@
   use_c10_dispatcher: full
   variants: function, method
 
+- func: moveaxis.intlist(Tensor(a) self, int[] source, int[] destination) -> Tensor(a)
```

Comment:
Add a comment before these registrations that moveaxis is an alias of movedim

Reply:
Done.